src/app/modules/group/components/group-join-by-code/group-join-by-code.component.html

```diff
@@ -23,14 +23,14 @@
       <span class="label red" *ngIf="!codeInfo.hasCodeNotSet && codeInfo.hasCodeExpired" i18n>
         This code has expired on {{ codeInfo.codeExpiration | date:'medium' }}
       </span>
-      <alg-button
-        label="Generate a code"
+      <p-button
+        i18n-label label="Generate a code"
         icon="fa fa-plus"
-        class="p-button-rounded"
-        (click)="generateNewCode()"
+        styleClass="alg-button p-button-rounded"
+        (onClick)="generateNewCode()"
         [disabled]="processing"
         *ngIf="codeInfo.hasCodeNotSet"
-      ></alg-button>
+      ></p-button>
```

Comment:
Is the style change intended?
Before:
![image](https://user-images.githubusercontent.com/16257159/168771463-16a7762e-71d0-499f-a633-7e16a57b3ba4.png)

After:
![image](https://user-images.githubusercontent.com/16257159/168771520-0eba9e35-6538-46ee-a232-ea1b45db7a2c.png)